sympy/geometry/point.py

```diff
@@ -30,13 +30,14 @@
 from sympy.core.numbers import Float
 from sympy.core.parameters import global_parameters
 from sympy.core.add import Add
+from sympy.core.evalf import EvalfMixin, prec_to_dps
 from sympy.utilities.iterables import uniq
 from sympy.utilities.misc import filldedent, func_name, Undecidable
 
 from .entity import GeometryEntity
 
 
-class Point(GeometryEntity):
+class Point(GeometryEntity, EvalfMixin):
```

Comment:
I think maybe it should be GeometryEntity that inherits `EvalfMixin` and then all subclasses should be fixed to make any `evalf`, `_eval_evalf` and `n` methods consistent.